packages/shared/providers.yaml

```diff
@@ -5339,3 +5339,27 @@ falai:
             type: string
             title: API Key
             description: The API key for your fal.ai account
+
+acuity-scheduling:
+    display_name: Acuity Scheduling
+    categories:
+        - scheduling
+        - appointments
```

Comment:
You can omit these two and have `productivity` only

Reply:
will do